
[2022-12-10T11:18:25.496] [ERROR] default - [ 'phone' ]
[2022-12-10T11:20:23.165] [ERROR] default - phone
[2022-12-10T11:22:15.870] [ERROR] default - phone
[2022-12-10T11:50:21.259] [ERROR] default - Error: Usuarios validation failed: phone: Path `phone` is required.
    at ValidationError.inspect (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\error\validation.js:50:26)
    at formatValue (node:internal/util/inspect:790:19)
    at inspect (node:internal/util/inspect:350:10)
    at formatWithOptionsInternal (node:internal/util/inspect:2241:40)
    at Object.format (node:internal/util/inspect:2098:10)
    at basicLayout (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\layouts.js:65:52)
    at app (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\appenders\file.js:98:23)
    at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\appenders\logLevelFilter.js:10:7
    at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\log4js.js:43:5
    at Array.forEach (<anonymous>) {
  errors: {
    phone: ValidatorError: Path `phone` is required.
        at validate (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\schematype.js:1346:13)
        at SchemaType.doValidate (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\schematype.js:1330:7)
        at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\document.js:2903:18
        at process.processTicksAndRejections (node:internal/process/task_queues:77:11) {
      properties: [Object],
      kind: 'required',
      path: 'phone',
      value: '+54 903816636350',
      reason: undefined,
      [Symbol(mongoose:validatorError)]: true
    }
  },
  _message: 'Usuarios validation failed'
}
[2022-12-10T11:51:37.766] [ERROR] default - Error: Usuarios validation failed: phone: Path `phone` is required.
    at ValidationError.inspect (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\error\validation.js:50:26)
    at formatValue (node:internal/util/inspect:790:19)
    at inspect (node:internal/util/inspect:350:10)
    at formatWithOptionsInternal (node:internal/util/inspect:2241:40)
    at Object.format (node:internal/util/inspect:2098:10)
    at basicLayout (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\layouts.js:65:52)
    at app (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\appenders\file.js:98:23)
    at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\appenders\logLevelFilter.js:10:7
    at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\log4js.js:43:5
    at Array.forEach (<anonymous>) {
  errors: {
    phone: ValidatorError: Path `phone` is required.
        at validate (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\schematype.js:1346:13)
        at SchemaType.doValidate (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\schematype.js:1330:7)
        at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\document.js:2903:18
        at process.processTicksAndRejections (node:internal/process/task_queues:77:11) {
      properties: [Object],
      kind: 'required',
      path: 'phone',
      value: '+54 903816636350',
      reason: undefined,
      [Symbol(mongoose:validatorError)]: true
    }
  },
  _message: 'Usuarios validation failed'
}
[2022-12-10T12:22:34.367] [ERROR] default - Error: Usuarios validation failed: phone: Path `phone` is required.
    at ValidationError.inspect (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\error\validation.js:50:26)
    at formatValue (node:internal/util/inspect:790:19)
    at inspect (node:internal/util/inspect:350:10)
    at formatWithOptionsInternal (node:internal/util/inspect:2241:40)
    at Object.format (node:internal/util/inspect:2098:10)
    at basicLayout (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\layouts.js:65:52)
    at app (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\appenders\file.js:98:23)
    at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\appenders\logLevelFilter.js:10:7
    at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\log4js.js:43:5
    at Array.forEach (<anonymous>) {
  errors: {
    phone: ValidatorError: Path `phone` is required.
        at validate (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\schematype.js:1346:13)
        at SchemaType.doValidate (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\schematype.js:1330:7)
        at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\document.js:2903:18
        at process.processTicksAndRejections (node:internal/process/task_queues:77:11) {
      properties: [Object],
      kind: 'required',
      path: 'phone',
      value: '+54 903816636350',
      reason: undefined,
      [Symbol(mongoose:validatorError)]: true
    }
  },
  _message: 'Usuarios validation failed'
}
[2022-12-10T12:25:21.563] [ERROR] default - Error: Usuarios validation failed: phone: Path `phone` is required.
    at ValidationError.inspect (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\error\validation.js:50:26)
    at formatValue (node:internal/util/inspect:790:19)
    at inspect (node:internal/util/inspect:350:10)
    at formatWithOptionsInternal (node:internal/util/inspect:2241:40)
    at Object.format (node:internal/util/inspect:2098:10)
    at basicLayout (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\layouts.js:65:52)
    at app (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\appenders\file.js:98:23)
    at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\appenders\logLevelFilter.js:10:7
    at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\log4js.js:43:5
    at Array.forEach (<anonymous>) {
  errors: {
    phone: ValidatorError: Path `phone` is required.
        at validate (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\schematype.js:1346:13)
        at SchemaType.doValidate (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\schematype.js:1330:7)
        at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\document.js:2903:18
        at process.processTicksAndRejections (node:internal/process/task_queues:77:11) {
      properties: [Object],
      kind: 'required',
      path: 'phone',
      value: '+54 903816636350',
      reason: undefined,
      [Symbol(mongoose:validatorError)]: true
    }
  },
  _message: 'Usuarios validation failed'
}
[2022-12-10T12:28:45.163] [ERROR] default - Error: Usuarios validation failed: phone: Path `phone` is required.
    at ValidationError.inspect (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\error\validation.js:50:26)
    at formatValue (node:internal/util/inspect:790:19)
    at inspect (node:internal/util/inspect:350:10)
    at formatWithOptionsInternal (node:internal/util/inspect:2241:40)
    at Object.format (node:internal/util/inspect:2098:10)
    at basicLayout (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\layouts.js:65:52)
    at app (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\appenders\file.js:98:23)
    at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\appenders\logLevelFilter.js:10:7
    at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\log4js.js:43:5
    at Array.forEach (<anonymous>) {
  errors: {
    phone: ValidatorError: Path `phone` is required.
        at validate (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\schematype.js:1346:13)
        at SchemaType.doValidate (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\schematype.js:1330:7)
        at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\document.js:2903:18
        at process.processTicksAndRejections (node:internal/process/task_queues:77:11) {
      properties: [Object],
      kind: 'required',
      path: 'phone',
      value: '+54 903816636350',
      reason: undefined,
      [Symbol(mongoose:validatorError)]: true
    }
  },
  _message: 'Usuarios validation failed'
}
[2022-12-10T12:31:41.905] [ERROR] default - Error: Usuarios validation failed: phone: Path `phone` is required.
    at ValidationError.inspect (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\error\validation.js:50:26)
    at formatValue (node:internal/util/inspect:790:19)
    at inspect (node:internal/util/inspect:350:10)
    at formatWithOptionsInternal (node:internal/util/inspect:2241:40)
    at Object.format (node:internal/util/inspect:2098:10)
    at basicLayout (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\layouts.js:65:52)
    at app (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\appenders\file.js:98:23)
    at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\appenders\logLevelFilter.js:10:7
    at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\log4js.js:43:5
    at Array.forEach (<anonymous>) {
  errors: {
    phone: ValidatorError: Path `phone` is required.
        at validate (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\schematype.js:1346:13)
        at SchemaType.doValidate (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\schematype.js:1330:7)
        at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\document.js:2903:18
        at process.processTicksAndRejections (node:internal/process/task_queues:77:11) {
      properties: [Object],
      kind: 'required',
      path: 'phone',
      value: '+54 903816636350',
      reason: undefined,
      [Symbol(mongoose:validatorError)]: true
    }
  },
  _message: 'Usuarios validation failed'
}
[2022-12-10T12:35:57.884] [ERROR] default - Error: Usuarios validation failed: phone: Path `phone` is required.
    at ValidationError.inspect (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\error\validation.js:50:26)
    at formatValue (node:internal/util/inspect:790:19)
    at inspect (node:internal/util/inspect:350:10)
    at formatWithOptionsInternal (node:internal/util/inspect:2241:40)
    at Object.format (node:internal/util/inspect:2098:10)
    at basicLayout (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\layouts.js:65:52)
    at app (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\appenders\file.js:98:23)
    at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\appenders\logLevelFilter.js:10:7
    at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\log4js.js:43:5
    at Array.forEach (<anonymous>) {
  errors: {
    phone: ValidatorError: Path `phone` is required.
        at validate (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\schematype.js:1346:13)
        at SchemaType.doValidate (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\schematype.js:1330:7)
        at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\document.js:2903:18
        at process.processTicksAndRejections (node:internal/process/task_queues:77:11) {
      properties: [Object],
      kind: 'required',
      path: 'phone',
      value: '+54 903816636350',
      reason: undefined,
      [Symbol(mongoose:validatorError)]: true
    }
  },
  _message: 'Usuarios validation failed'
}
[2022-12-10T13:21:36.740] [ERROR] default - Error: Usuarios validation failed: phone: Path `phone` is required.
    at ValidationError.inspect (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\error\validation.js:50:26)
    at formatValue (node:internal/util/inspect:790:19)
    at inspect (node:internal/util/inspect:350:10)
    at formatWithOptionsInternal (node:internal/util/inspect:2241:40)
    at Object.format (node:internal/util/inspect:2098:10)
    at basicLayout (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\layouts.js:65:52)
    at app (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\appenders\file.js:98:23)
    at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\appenders\logLevelFilter.js:10:7
    at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\log4js.js:43:5
    at Array.forEach (<anonymous>) {
  errors: {
    phone: ValidatorError: Path `phone` is required.
        at validate (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\schematype.js:1346:13)
        at SchemaType.doValidate (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\schematype.js:1330:7)
        at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\document.js:2903:18
        at process.processTicksAndRejections (node:internal/process/task_queues:77:11) {
      properties: [Object],
      kind: 'required',
      path: 'phone',
      value: '+54 903816636350',
      reason: undefined,
      [Symbol(mongoose:validatorError)]: true
    }
  },
  _message: 'Usuarios validation failed'
}
[2022-12-10T13:22:50.751] [ERROR] default - Error: Usuarios validation failed: phone: Path `phone` is required.
    at ValidationError.inspect (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\error\validation.js:50:26)
    at formatValue (node:internal/util/inspect:790:19)
    at inspect (node:internal/util/inspect:350:10)
    at formatWithOptionsInternal (node:internal/util/inspect:2241:40)
    at Object.format (node:internal/util/inspect:2098:10)
    at basicLayout (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\layouts.js:65:52)
    at app (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\appenders\file.js:98:23)
    at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\appenders\logLevelFilter.js:10:7
    at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\log4js\lib\log4js.js:43:5
    at Array.forEach (<anonymous>) {
  errors: {
    phone: ValidatorError: Path `phone` is required.
        at validate (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\schematype.js:1346:13)
        at SchemaType.doValidate (C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\schematype.js:1330:7)
        at C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\node_modules\mongoose\lib\document.js:2903:18
        at process.processTicksAndRejections (node:internal/process/task_queues:77:11) {
      properties: [Object],
      kind: 'required',
      path: 'phone',
      value: '+54 903816636350',
      reason: undefined,
      [Symbol(mongoose:validatorError)]: true
    }
  },
  _message: 'Usuarios validation failed'
}
[2022-12-10T14:00:26.246] [ERROR] default - [Error: EISDIR: illegal operation on a directory, read] {
  errno: -4068,
  code: 'ESTREAM',
  syscall: 'read',
  command: 'API'
}
[2022-12-10T14:02:00.691] [ERROR] default - [Error: ENOENT: no such file or directory, open 'C:\public\uploads'] {
  errno: -4058,
  code: 'ESTREAM',
  syscall: 'open',
  path: 'C:\\public\\uploads',
  command: 'API'
}
[2022-12-10T14:04:16.336] [ERROR] default - [Error: ENOENT: no such file or directory, open 'C:\public\uploads\1670691855253perfil.jpg'] {
  errno: -4058,
  code: 'ESTREAM',
  syscall: 'open',
  path: 'C:\\public\\uploads\\1670691855253perfil.jpg',
  command: 'API'
}
[2022-12-10T14:07:17.955] [ERROR] default - [Error: ENOENT: no such file or directory, open 'C:\uploads\1670692036817perfil.jpg'] {
  errno: -4058,
  code: 'ESTREAM',
  syscall: 'open',
  path: 'C:\\uploads\\1670692036817perfil.jpg',
  command: 'API'
}
[2022-12-10T16:47:05.487] [ERROR] default - [Error: ENOENT: no such file or directory, open 'C:\uploads'] {
  errno: -4058,
  code: 'ESTREAM',
  syscall: 'open',
  path: 'C:\\uploads',
  command: 'API'
}
[2022-12-10T16:48:26.583] [ERROR] default - [Error: ENOENT: no such file or directory, open 'C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\file:\C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\src\utils\nodemailer.js\uploads'] {
  errno: -4058,
  code: 'ESTREAM',
  syscall: 'open',
  path: 'C:\\Users\\sebas\\Dropbox\\PROGRAMACION\\CODERHOUSE\\FULLSTACK\\4.BackEnd\\42.API-REST\\file:\\C:\\Users\\sebas\\Dropbox\\PROGRAMACION\\CODERHOUSE\\FULLSTACK\\4.BackEnd\\42.API-REST\\src\\utils\\nodemailer.js\\uploads',
  command: 'API'
}
[2022-12-10T16:49:42.461] [ERROR] default - [Error: ENOENT: no such file or directory, open 'C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\file:\C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\src\utils\nodemailer.js'] {
  errno: -4058,
  code: 'ESTREAM',
  syscall: 'open',
  path: 'C:\\Users\\sebas\\Dropbox\\PROGRAMACION\\CODERHOUSE\\FULLSTACK\\4.BackEnd\\42.API-REST\\file:\\C:\\Users\\sebas\\Dropbox\\PROGRAMACION\\CODERHOUSE\\FULLSTACK\\4.BackEnd\\42.API-REST\\src\\utils\\nodemailer.js',
  command: 'API'
}
[2022-12-10T16:58:21.839] [ERROR] default - [Error: EISDIR: illegal operation on a directory, read] {
  errno: -4068,
  code: 'ESTREAM',
  syscall: 'read',
  command: 'API'
}
[2022-12-10T17:07:47.280] [ERROR] default - [Error: ENOENT: no such file or directory, open 'C:\uploads'] {
  errno: -4058,
  code: 'ESTREAM',
  syscall: 'open',
  path: 'C:\\uploads',
  command: 'API'
}
[2022-12-10T17:20:37.441] [ERROR] default - [Error: ENOENT: no such file or directory, open 'C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\uploads'] {
  errno: -4058,
  code: 'ESTREAM',
  syscall: 'open',
  path: 'C:\\Users\\sebas\\Dropbox\\PROGRAMACION\\CODERHOUSE\\FULLSTACK\\4.BackEnd\\42.API-REST\\uploads',
  command: 'API'
}
[2022-12-10T17:21:12.144] [ERROR] default - [Error: ENOENT: no such file or directory, open 'C:\Users\sebas\Dropbox\PROGRAMACION\CODERHOUSE\FULLSTACK\4.BackEnd\42.API-REST\uploads'] {
  errno: -4058,
  code: 'ESTREAM',
  syscall: 'open',
  path: 'C:\\Users\\sebas\\Dropbox\\PROGRAMACION\\CODERHOUSE\\FULLSTACK\\4.BackEnd\\42.API-REST\\uploads',
  command: 'API'
}
[2022-12-11T10:47:13.570] [FATAL] default - Error en servidor: Error: listen EADDRINUSE: address already in use :::8080
